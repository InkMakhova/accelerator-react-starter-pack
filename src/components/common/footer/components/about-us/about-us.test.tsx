import React from 'react';
import { render } from '@testing-library/react';
import AboutUs from './about-us';

it('should render AboutUs correctly', () => {
  const {container} = render(<AboutUs />);
  expect(container).toMatchSnapshot();
});
