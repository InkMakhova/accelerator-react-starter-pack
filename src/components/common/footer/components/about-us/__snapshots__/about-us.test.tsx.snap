// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AboutUs correctly 1`] = `
<div>
  <section
    class="footer__nav-section footer__nav-section--info"
  >
    <h2
      class="footer__nav-title"
    >
      О нас
    </h2>
    <p
      class="footer__nav-content footer__nav-content--font-secondary"
    >
      Магазин гитар, музыкальных инструментов и гитарная мастерская 
      <br />
       в Санкт-Петербурге.
      <br />
      <br />
      Все инструменты проверены, отстроены 
      <br />
       и доведены до идеала!
    </p>
  </section>
</div>
`;
